api/rates.go

```diff
@@ -31,7 +31,7 @@ func (r Rates) Sort() {
 // Current returns the rates current rate or error
 func (r Rates) Current(now time.Time) (Rate, error) {
 	for _, rr := range r {
-		if (rr.Start.Before(now) || rr.Start.Equal(now)) && rr.End.After(now) {
+		if !rr.Start.After(now) && rr.End.After(now) {
```

Comment:
Das macht bei mir einen Knoten im Kopf. Wieso nicht:

```suggestion
		if rr.Start.Before(now) && rr.End.After(now) {
```

Reply:
Weil es ein halboffenes Intervall ist!